pkgs/development/libraries/libxcrypt/default.nix

```diff
@@ -15,6 +15,11 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-5eH0yu4KAd4q7ibjE4gH1tPKK45nKHlm0f79ZeH9iUM=";
   };
 
+  postPatch = ''
+    # this could be accomplished by updateAutotoolsGnuConfigScriptsHook, but that causes infinite recursion
+    substituteInPlace ./build-aux/m4-autogen/config.guess --replace-fail /usr/bin/uname uname
+  '';
```

Comment:
```suggestion
  # this could be accomplished by updateAutotoolsGnuConfigScriptsHook, but that causes infinite recursion
  postPatch = ''
    substituteInPlace ./build-aux/m4-autogen/config.guess --replace-fail /usr/bin/uname uname
  '';
```